code/modules/mob/living/simple_animal/hostile/megafauna/dragon.dm

```diff
@@ -30,13 +30,18 @@
 	var/obj/item/device/gps/internal
 	var/swooping = 0
 	var/swoop_cooldown = 0
-	deathmessage = "collapes into a pile of bones, it's flesh sloughing away."
+	deathmessage = "collapses into a pile of bones, its flesh sloughing away."
 	death_sound = 'sound/magic/demon_dies.ogg'
 	damage_coeff = list(BRUTE = 1, BURN = 0.5, TOX = 1, CLONE = 1, STAMINA = 0, OXY = 1)
 
 /mob/living/simple_animal/hostile/megafauna/dragon/New()
 	..()
 	internal = new/obj/item/device/gps/internal/dragon(src)
+	deathmessage = "[src] [deathmessage]"
```

Comment:
I was more wondering why it wasn't done like this https://github.com/tgstation/-tg-station/pull/17761